From 6a007cbc00f860d6857a3271948ca0564f93e1fe Mon Sep 17 00:00:00 2001
From: Dmitri Smirnov <d.s@plexoos.com>
Date: Mon, 19 Aug 2019 17:50:11 -0400
Subject: [PATCH 3/4] gcc5.4 complains about undefined fpos symbol

---
 StarVMC/StarAgmlLib/AgMath.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/StarVMC/StarAgmlLib/AgMath.h b/StarVMC/StarAgmlLib/AgMath.h
index 54138f1a55..d57b8eb554 100644
--- a/StarVMC/StarAgmlLib/AgMath.h
+++ b/StarVMC/StarAgmlLib/AgMath.h
@@ -39,7 +39,6 @@ const Double_t keV    = 0.001 * MeV;
 #define dup _dup_
 
 #define tanf _tanf_
-#define fpos _fpos_
 //
 // Implement some bitwise functions from fortran land
 //
-- 
2.16.1

